HW1/task2.py

```diff
@@ -0,0 +1,18 @@
+# Петя, Катя и Сережа делают из бумаги журавликов. 
+# Вместе они сделали S журавликов.
+#  Сколько журавликов сделал каждый ребенок, если известно,
+#  что Петя и Сережа сделали одинаковое количество журавликов,
+#  а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+
+s = int(input("Введите число, кратное 6:")) 
+
+if s // 6 :
+    print("Введено неверное число, надо ввести число кратное 6")
```

Comment:
Проверка на кратность осуществляется несколько иначе.
```suggestion
if s % 6 == 0 :
    print("Введено неверное число, надо ввести число кратное 6")
```